src/test/scala/org/constellation/api/ConfigEndpointsApiTest.scala

```diff
@@ -36,18 +36,28 @@ class TimeoutEndpointApiTest
     api = new API()(system, Timeout(10, TimeUnit.SECONDS), dao)
   }
 
-  "Timeout Endpoints" - {
-    "should get current value simulateEndpointTimeout variable" in {
-      Get("/timeout") ~> api.getEndpoints ~> check {
+  "SimulateTimeout Endpoints" - {
+    "should execute method to change value simulateEndpointTimeout variable" in {
+      Post("/timeout") ~> api.configEndpoints ~> check {
```

Comment:
AFAIK there should be one assertion per test case, so imo it should be:
`should allow to enable simulating timeouts` and `should allow to disable simulating timeouts`
and the same with `should allow to enable random transactions` / `should allow to disable random transactions`

Reply:
Okey I thought the test boundary is check block